### DISCUSSION TITLE
Run JavaScript before page is fully loaded

### DISCUSSION THREAD

**Discussion by edwyn-amador** [AutomaApp/automa#709] [Q&A] [answered]
How can I run JS before the page loads?

Before I was able to do it without problem, it was perfect because I was able to apply changes immediately and those changes requiring waiting for Dom to be ready, I simply create a listener event for that, but recently it seems that it is hardcoded to the DOMContentLoaded event...

How can I make it work like it was before, so I apply my JS code immediately?

**Comment by edwyn-amador** [comment]
@Kholid060  I really need a way to execute JS before the page loads, I've been looking if there' any checkbox/toggle button for this but haven't find any... if this isn't possible anymore can you just implement a setting under the JS block to allow this, please? Thanks

**Comment by Kholid060** [comment, marked as answer]
Sure, I'll add an option to insert the JS before the page loads on the next update.

**Reply by edwyn-amador** [reply]
Thanks @Kholid060 , I appreciate it!

**Reply by edwyn-amador** [reply]
@Kholid060 Thanks for adding this! Now my JS code applies immediately :)
